draft-bbs-signatures.md

```diff
@@ -527,119 +583,153 @@ Procedure:
 
 3. (A, e, s) = signature_result
 
-4. (i1, i2,..., iR) = RevealedIndexes
+4. gen_octets =  (H_s || H_d || H_1 || ... || H_L)
 
-5. (j1, j2,..., jU) = [L] \ RevealedIndexes
+5. domain_prime = (PK || L || gen_octets || Ciphersuite_ID || header)
```

Comment:
It's less clear now that the hash input encoding rules apply here. I think we need to define a helper function like `encode_for_hash` and explain the encoding rules there. That could be a separate update, though.

Reply:
That's true. I would prefer taking this up in another PR as well though. Opened #185 to keep track of this. Thanks for raising.